packages/blockchain-wallet-v4-frontend/src/components/Form/TextAreaDebounced/index.js

```diff
@@ -26,7 +26,7 @@ const getErrorState = (meta) => {
 class TextAreaDebounced extends React.Component {
   constructor (props) {
     super(props)
-    this.state = { value: props.input.value }
+    this.state = { value: props.input.value, updatedValue: props.input.value }
```

Comment:
Where is updatedValue used?

Reply:
It's used right here @plondon https://github.com/jstettner/blockchain-wallet-v4-frontend/blob/85283eaf3ae4604402e9e9b148c15ecf53933735/packages/blockchain-wallet-v4-frontend/src/components/Form/TextAreaDebounced/index.js#L37